components/resourceDetails/resourceSharing/agentAccess/agentAccessOptionsMenu/index.test.jsx

```diff
@@ -95,14 +95,15 @@ describe("Access Details Modal", () => {
     });
   });
 
-  it("closes the details modal when you click on the revoke button", async () => {
+  // FIXME: find out why this test suddenly fails
```

Comment:
Shouldn't we fix this before we merge?

Reply:
Yes, I just wanted it to build for testing.

Reply:
fixed.